### DISCUSSION TITLE
How to have a persistent marker only draw an indication on ONE series of data in 2.0.0-alpha.27

### DISCUSSION THREAD

**Discussion by Tyler-Lopez** [patrykandpatrick/vico#857] [Questions] [answered]
### Question

The latest update `2.0.0-alpha.27` broke my solution to having a persistent marker with a label only on the most recently added data layer. 

I used to use `override fun draw(context: CartesianDrawContext, targets: List<CartesianMarker.Target>)` to simply update the list of targets to only include the last target. Now that is not exposed. Was this considered or is there a better way to do this?

What is exposed now is just `override fun CartesianDrawingContext.drawIndicator(x: Float, y: Float, color: Int, halfIndicatorSize: Float)`

### Vico version(s)

2.0.0-alpha.27

### UI framework(s)

Jetpack Compose

**Comment by Tyler-Lopez** [comment, marked as answer]
🤦 It is still exposed, my bad! For whatever reason, my IDE, was failing me with the `CartesianDrawContext` --> `CartesianDrawingContext`.

**Reply by patrickmichalik** [reply]
I’m glad to hear that you’ve solved this. Thanks for following up!
